### PR TITLE
Update deprecated Array constructor

### PR DESCRIPTION
**Pull request by musm**

### CODE REVIEW

File: src/LibExpat.jl [hunk 1]
```diff
@@ -298,7 +298,7 @@ function find{T<:AbstractString}(pd::ETree, path::T)
         end
     end
 
-    xp= Array(Tuple{Symbol,Any},0)
+    xp= Array{Tuple{Symbol,Any}}(0)
```
Comment (SimonDanisch) [inline src/LibExpat.jl]: Should we turn this rather into:
```Julia
Tuple{Symbol,Any}[]
```

Comment (musm) [inline src/LibExpat.jl]: The other methods could also be instead written as `SymbolAny[]` and `returntype[]` Is that easier to read that the explicit array constructor? 

(fyi can you turn on branch protection for master, to be safe? the move  in the org and my fork being based on the amit's repo kinda screwed things up)

Comment (SimonDanisch) [inline src/LibExpat.jl]: Maybe it's just me, but I trip over the `(0)`.  I find it much clearer, that [] is the empty Vector ;) Besides, it's less typing :D

Comment (musm) [inline src/LibExpat.jl]: I think appveyor also needs to be configured and also attobot

Comment (SimonDanisch) [inline src/LibExpat.jl]: branch protection sounds like a good idea :) I turned it on!

**Comment by musm** [comment]
@SimonDanisch  does that look easier to read for you?

**Comment by SimonDanisch** [comment]
@musm almost too good to read :) :P

**Pull request merged**